RELEASE.md

```diff
@@ -0,0 +1,256 @@
+   ```
+
+   b. Push this branch to the upstream repo:
+   git push origin-aws
+
+    1. **If it’s only a patch version and the branch already exists**, checkout the existing branch (e.g. git checkout origin-aws/v0.2), and merge in the commit you’ve merge into main with the updates to the CHANGELOG.md file.
+    2. **Attributions file (ORT)**: Make sure the attributions file is updated by manually running the ORT workflow:
+
+        a. Go to: https://github.com/valkey-io/valkey-glide/actions/workflows/ort.yml <br>
+
+        b. On the right corner, click on “Run workflow” and set the branch to run against the ORT tool to your newly created branch name (e.g. v0.3), and the exact release version (e.g. 0.3.0) <br>
+
+        c. <br>
+        ![OSS Review Toolkit](images-release-md/oss-review-toolkit.png)
+
+        d. If diff is found the workflow will open a PR in the release branch with the required changes. Make sure that all dependencies are approved by a Valkey-GLIDE maintainer, refer to the SOP - Check the Licenses: ORT/Attributions File Setup and Usage Instructions. Merge the PR once it is ready.
+
+        1. Trigger the CD workflow
+
+            a. The CD workflow will deploy the code into the package managers of the different wrappers. If there is any issue with the CD, **contact a Valkey-GLIDE maintainer**.
+
+            b. To trigger a new release we need to tag the new branch commit with the release version, if its a release candidate, which should
+                   be release before a real release in order to test the version, add
+                   -rc$RC_VERSION:
+
+            c. git tag v1.x.x-rc0
+
+            d. Find how you saved the upstream remote repo (valkey-io/valkey-glide):
+
+            e.
+
+            ```
+            [10:09:23] ubuntu (v1.x*)$ git remote -vv
+            origin  https://github.com/your-repo/valkey-glide (fetch)
+            origin  https://github.com/your-repo/valkey-glide (push)
+            origin-aws      https://github.com/valkey-io/valkey-glide (fetch)
+            origin-aws      https://github.com/valkey-io/valkey-glide (push)
+            ```
+
+            f. Push the tag to the upstream remote so it would trigger the CD action:
+
+            g. git push origin-aws v1.x.x-rc0
+
+            h. For each time your release fails all you need to do is create new tag in which the next rc version (e.g. v0.5.0-rc3 → v0.5.0-rc4).
+
+            i. After the release candidate deployed, in node tests running automatically and when release candidate CD workflow is done successfully we can immediately release the real version. In python there’s no such support currently, so you'll first need to test the RC version you deployed on the different supported platform.You can do so by cloning the repo into each machine, go to [examples/python/requirements.txt](https://github.com/valkey-io/valkey-glide/blob/main/examples/python/requirements.txt).
```

Comment:
Don't mess up how to test an RC for node and for python clients.
Add a new titles for each client and describe all there